packages/formatting/presets/strict.yaml

```
@@ -0,0 +1,53 @@
+# @outfitter/formatting Strict Preset
+# Rigorous formatting for documentation-heavy projects
+name: strict
+description: Strict formatting rules optimized for documentation and long-term maintenance
+extends: standard  # Inherit from standard and override specific settings
+
+common:
+  # Stricter line length
+  lineWidth: 80
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)